pkg/sgxcontroller/sgxcontroller.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sgxcontroller
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/intel/intel-device-plugins-for-kubernetes/pkg/internal/containers"
+)
+
+// +kubebuilder:webhook:path=/pods-sgx,mutating=true,failurePolicy=fail,groups="",resources=pods,verbs=create;update,versions=v1,name=sgx.mutator.webhooks.intel.com,sideEffects=NoneOnDryRun
```

Comment:
`failurePolicy` here is not the same as in the corresponding CRD. The CRD value is going to be overriden with `make generate`.

Reply:
that was an oversight from my side, thanks for noticing!